Dockerfile

```diff
@@ -48,6 +48,10 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
 #  && npm install -g karma-firefox-launcher
 
 COPY runtime/tests/google-chrome.repo /etc/yum.repos.d/google-chrome.repo
+
+# following downgrade is needed as GKE build does not install latest chrome.
+# Functional tests do pass on latest chrome version-65 (as of this writing)
+# Replace following 2 lines by following whenever you want to upgrade to latest
```

Comment:
Typo here. It should be
`Remove the following 2 lines whenever you wish to upgrade to the latest version of chrome`